sdk/communication/azure-communication-email/src/main/java/com/azure/communication/email/implementation/models/EmailAttachment.java

```diff
@@ -41,20 +41,20 @@ public final class EmailAttachment implements JsonSerializable<EmailAttachment>
 
     /**
      * Creates an instance of EmailAttachment class.
-     *
+     * 
      * @param name the name value to set.
      * @param contentType the contentType value to set.
      * @param contentInBase64 the contentInBase64 value to set.
      */
-    public EmailAttachment(String name, String contentType, BinaryData contentInBase64) {
+    public EmailAttachment(String name, String contentType, byte[] contentInBase64) {
```

Comment:
Will this not break the code in [EmailAsyncClient](https://github.com/Azure/azure-sdk-for-java/blob/679831e7835ab0a83e071032d73086762f54e0d4/sdk/communication/azure-communication-email/src/main/java/com/azure/communication/email/EmailAsyncClient.java#L95)?

Reply:
Seems Autorest changes somehow here.